content/blog/2017/2017-04-10-Jenkins-has-upgraded-to-Java-8.adoc

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: "Jenkins Now Requires Java 8 To Run Starting With 2.54"
+tags:
+- java8
+- upgrade
+author: batmat
+---
+
+:imagesdir: /images/post-images/2017-04-java8
+
+image:lambda.png[role="right"] As link:/blog/2017/01/17/Jenkins-is-upgrading-to-Java-8/[announced some weeks ago], starting with version link:https://jenkins.io/changelog/#v2.54[Jenkins **2.54**], the Java runtime requirement has been updated to Java 8.
+
+== Timeline
+
+* link:/blog/2017/01/17/Jenkins-is-upgrading-to-Java-8/[Earlier this year: announce this upgrade].
+* April, 2017: Drop support for Java 7 in Jenkins weekly: **2.54**.
+* June, 2017: First LTS version requiring Java 8 is published.
+
+== FAQ
+
+I have only a JRE 7 available, can I still run Jenkins 2.54+ with it?::
+No, you need to install a JRE 8.
+
+Can I still run my agents with Java 7?::
+No. An agent must run at least the Java version which the Jenkins Master was compiled for.
+This means you must also upgrade your agents JVM if they are still running Java 7.
+
```

Comment:
Follow up to https://issues.jenkins-ci.org/browse/JENKINS-40990. Add:

> Can I run Maven projects with Java 7?

Correct behavior is not guaranteed. Proceed **at your own risk**.
Maven Project uses Jenkins Remoting to establish interceptors within the Maven executable. For this purpose Maven uses Remoting and some Jenkins core classes, hence the behavior may break with any update.